GETTING_STARTED.md

```diff
@@ -0,0 +1,166 @@
+
+The next thing we want to do is define where our packages live. We do this by adding a `workspaces` field to our root `package.json`. For our project, we are going to say that packages can live in three locations:
+
+- A `/apps` folder for blah blah blah
+- A `/scripts` folder for blah blah blah
+- A `/packages` folder for... packages...
+
+We only want to allow folders at the first level of each of these to be checked for being packages, so we want a workspaces field that looks like:
+
+```json
+  "workspaces": [
+    "packages/*",
+    "apps/*",
+    "scripts/*"
+  ]
+```
+
+Which should leave our root `package.json` looking like:
+
+```json
+{
+  "name": "@monorepo-starter/root",
+  "version": "1.0.0",
+  "private": true,
+  "workspaces": ["packages/*", "apps/*", "scripts/*"]
+}
+```
+
+With that set up, you can now run `yarn` as your install script, and it will install all dependencies of all packages within those folders.
+
+Next up, let's add some packages so we can have some dependencies!
+
+### Your package folders
+
+For this tutorial, we are going to be setting up 5 packages. They will be:
+
+- `@monorepo-starter/simple-package`
+- `@monorepo-starter/private-package`
+- `@monorepo-starter/with-multiple-entrypoints`
+- `@monorepo-starter/placeholder-website`
+- `@monorepo-starter/placeholder-app`
+
+We are going to learn something different while setting up each of these packages. We will set up the `simple-package`, and `placeholder-website` now, and `with-multiple-entrypoints`, and `placeholder-app` a little later when we set up our build tooling. Finally, we will add our `private-package` when we set up publishing.
+
+#### Adding `@monorepo-starter/simple-package`
+
+This package is going to be our bedrock simplest package.
+
+```shell
+mkdir packages
+mkdir packages/simple-package
+```
+
+In our newly created `simple-package` directory, add the following `package.json`:
+
+```json
+{
+  "name": "@monorepo-starter/simple-package",
+  "version": "1.0.0",
+  "description": "A very simple package within a monorepo"
+}
+```
+
+We want our package to do something, so we are going to add a simple function that the package exports, which takes in a string, and outputs nice colourful letters using [cfonts](https://www.npmjs.com/package/cfonts)
+
+The first thing we will want to do is install `cfonts`.
+
+To do this, we will want to run:
+
+```
+cd packages/simple-package
+yarn add cfonts
+```
+
+This adds `cfonts` as a dependency to the `simple-package`, and installs it. Note the folder structure here:
+
+> TODO images of the install of this
+
+## Setting up building your packages
+
+```shell
+yarn add @preconstruct/cli
+yarn preconstruct init
+```
+
+Add scripts to your `package.json`
+
+Add `preconstruct` field to the root package.json
+
+```json
+"scripts": {
+"postinstall": "preconstruct dev",
+"build": "preconstruct build",
+}
+```
+
+> TODO teach preconstruct fix also
+
+## Adding Manypkg to help validate your packages
```

Comment:
But why? what does "help validate your packages" mean? 

-> help validate that your packages are installed correctly 
or some such, instead?

Reply:
We don't have a single sentence 'why you need manypkg' as the answer is basically 'there is a host of incredible pain that manypkg wards off that we hope you never need to understand'. Working on it.